content/v2/domains.markdown

```diff
@@ -52,6 +52,29 @@ List all domains for the account 1010 that have name matching "example":
           -H 'Accept: application/json' \
           https://api.dnsimple.com/v2/1010/domains?name_like=example
 
+### Sorting
+
+Sort records with the `sorting` param by specifing the field and the sorting policy: `asc` for ascending and `desc` for descending.
+Multiple fields are allowed and MUST be comma separated.
```

Comment:
What about:

> Multiple fields are allowed and _MUST_ be comma separated. The order of fields *IS* relevant, as it will determine which order criteria takes priority.

We can add an example with 3 tuples and 2 different orderings to show how the order affects the result.